core-build/gulp-core-build/src/logging.ts

```diff
@@ -836,7 +843,14 @@ export function markTaskCreationTime(): void {
 }
 
 function messageIsSuppressed(message: string): boolean {
-  return localCache.errorAndWarningSuppressions[normalizeMessage(message)];
+  for (const suppression of localCache.errorAndWarningSuppressions) {
+    if (typeof suppression === 'string' && message === suppression) {
+      return true;
+    } else if (suppression instanceof RegExp && message.match(suppression)) {
+      return true;
+    }
+  }
+  return false;
```

Comment:
You could simplify this with a `for(let i: number = 0; ... ` loop if you want to.